boost-maven/boost-maven-plugin/src/main/java/io/openliberty/boost/maven/liberty/LibertyPackageMojo.java

```diff
@@ -333,6 +299,31 @@ private void installMissingFeatures() throws MojoExecutionException {
         executeMojo(getPlugin(), goal("install-feature"), configuration(element(name("serverName"), libertyServerName),
                 element(name("features"), element(name("acceptLicense"), "false"))), getExecutionEnvironment());
     }
+    
+    /**
+     * Get all booster dependencies and invoke the maven-dependency-plugin 
+     * to copy them to the Liberty server.
+     * @throws MojoExecutionException 
+     *
+     */
+    private void copyBoosterDependencies() throws MojoExecutionException {
+    	
+    	List<String> dependenciesToCopy = boosterUtil.getDependenciesToCopy();
+    	
+    		
+		for (String dep :  dependenciesToCopy) {
+			
+			String[] dependencyInfo = dep.split(":");
+			
+			executeMojo(getMavenDependencyPlugin(), goal("copy"), configuration(element(name("outputDirectory"), libertyServerPath + "/resources"),
+	    			element(name("artifactItems"), 
```

Comment:
For Maven and Gradle... is  the server resources directory better .. or the shared resources dir?     Perhaps it doesn't matter since this is targeted to a single server... but we still put the SpringBoot dependencies into shared... IDK.

Reply:
I don't think `server package` will package up what's in the shared resource dir into the uber jar.

Reply:
Didn't realize that.